tests/unit/support/queue.ts

```diff
@@ -166,5 +167,51 @@ registerSuite({
 		setTimeout(dfd.callback(function () {
 			assert.equal(parts.join(','), 'start,end');
 		}), 100);
+	},
+
+	'web workers': {
+		'queue from webworker': function (this: any) {
+			if (global.Worker === undefined) {
+				this.skip('does not support web workers');
+				return;
+			}
+
+			const baseUrl = location.origin;
+			const dfd = this.async(10000);
+			const blob = new Blob([ `(function() { 
```

Comment:
Android 4.4.4 supports WebWorkers but doesn't support [Blob creation this way](https://caniuse.com/#feat=blobbuilder).  Again, not a big deal, but maybe add a simply blob check too?

Also, this might be a good pattern for doing these tests (thank you!) and might want to consider opening an issue on test-extras to add this as some sort of helper.